### PR TITLE
LPS-182766 | Move Modal in Knowledge Base

### PR DESCRIPTION
**Pull request by jaimelr10**
Resend from: https://github.com/liferay-lima/liferay-portal/pull/4528
Change names because of https://github.com/brianchandotcom/liferay-portal/pull/138451#issuecomment-1660285519

Description: We need to provide the user a way to identify which and where is moving an element when using the new move modal for knowledge base.

Jira Ticket: https://issues.liferay.com/browse/LPS-182766

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_kb_object_modal.jsp [hunk 1]
```diff
@@ -19,7 +19,7 @@ KBAdminNavigationDisplayContext kbAdminNavigationDisplayContext = new KBAdminNav
 			HashMapBuilder.<String, Object>put(
 				"items", kbAdminNavigationDisplayContext.getKBFolderDataJSONArray()
 			).put(
-				"itemToMoveParentId", kbAdminNavigationDisplayContext.getKBObjectToMoveParentId()
+				"moveParentItemId", kbAdminNavigationDisplayContext.getMoveParentKBObjectId()
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_kb_object_modal.jsp:22]: There is a reason to not call `moveParentKBObjectId` instead `moveParentItemId`
Usually is :
**name** = get**Name**

Comment (jaimelr10) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_kb_object_modal.jsp:22]: No, I'll fix it! :)

**Review by ambrinchaudhary — APPROVED** [review]
I don't see any issues with the variable names... hope @AliciaGarciaGarcia thinks the same

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by austinchiang** [comment]
This test failure looks relevant: https://testray.liferay.com/home/-/testray/case_results/2399433314
`beanshell] 2023-08-03 12:43:35.932 ERROR [com.liferay.knowledge.base.test-executor-thread][GroupLocalServiceImpl:4136] Unable to delete data for portlet null in group 51746
[beanshell] com.liferay.exportimport.kernel.lar.PortletDataException: null
[beanshell] 	at com.liferay.exportimport.kernel.lar.BasePortletDataHandler._handleException(BasePortletDataHandler.java:801) ~[portal-kernel.jar:?]
[beanshell] 	at com.liferay.exportimport.kernel.lar.BasePortletDataHandler.deleteData(BasePortletDataHandler.java:91) ~[portal-kernel.jar:?]
[beanshell] 	at com.liferay.portal.service.impl.GroupLocalServiceImpl.deletePortletData(GroupLocalServiceImpl.java:4131) [portal-impl.jar:?]
[beanshell] 	at com.liferay.portal.service.impl.GroupLocalServiceImpl.deleteGroup(GroupLocalServiceImpl.java:1040) [portal-impl.jar:?]
[beanshell] 	at sun.reflect.GeneratedMethodAccessor3259.invoke(Unknown Source) ~[?:?]
[beanshell] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_121]
[beanshell] 	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_121]
[beanshell] 	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:41) [portal-impl.jar:?]
[beanshell] 	at com.liferay.portal.spring.transaction.TransactionInterceptor.invoke(TransactionInterceptor.java:60) [portal-impl.jar:?]
[beanshell] 	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:48) [portal-impl.jar:?]
[beanshell] 	at com.liferay.portal.kernel.aop.ChainableMethodAdvice.invoke(ChainableMethodAdvice.java:55) [portal-kernel.jar:?]
[beanshell] 	at com.liferay.portal.spring.aop.AopMethodInvocationImpl.proceed(AopMethodInvocationImpl.java:48) [portal-impl.jar:?]
[beanshell] 	at com.liferay.portal.spring.aop.AopInvocationHandler.invoke(AopInvocationHandler.java:40) [portal-impl.jar:?]
[beanshell] 	at com.sun.proxy.$Proxy109.deleteGroup(Unknown Source) [?:?]
[beanshell] 	at sun.reflect.GeneratedMethodAccessor3258.invoke(Unknown Source) ~[?:?]
[beanshell] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_121]
[beanshell] 	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_121]
[beanshell] 	at com.liferay.portal.kernel.test.rule.DataGuardTestRuleUtil.smartDelete(DataGuardTestRuleUtil.java:170) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.DataGuardTestRuleUtil._autoDeleteLeftovers(DataGuardTestRuleUtil.java:384) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.DataGuardTestRuleUtil._autoDeleteAndAssert(DataGuardTestRuleUtil.java:269) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.DataGuardTestRuleUtil.afterClass(DataGuardTestRuleUtil.java:89) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.DataGuardTestRule.afterClass(DataGuardTestRule.java:28) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.DataGuardTestRule.afterClass(DataGuardTestRule.java:13) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:57) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:50) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:50) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:50) [bundleFile:?]
[beanshell] 	at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:50) [bundleFile:?]
[beanshell] 	at org.junit.rules.RunRules.evaluate(RunRules.java:20) [bundleFile:?]
[beanshell] 	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable._execute(TestExecutorRunnable.java:307) [bundleFile:?]
[beanshell] 	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable.run(TestExecutorRunnable.java:86) [bundleFile:?]
[beanshell] 	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
[beanshell] Caused by: java.lang.NullPointerException
[beanshell] 	at com.liferay.portal.service.impl.RoleLocalServiceImpl.getDefaultGroupRole(RoleLocalServiceImpl.java:705) ~[portal-impl.jar:?]`

@jaimelr10 @alicia @ambrinchaudhary

**Comment by austinchiang** [comment]
Actually I just found this was failing in https://testray.liferay.com/home/-/testray/case_results/2400198934 as well

**Comment by AliciaGarciaGarcia** [comment]
Hi @austinchiang , I think that the error is the same than here https://liferay.atlassian.net/browse/LPS-186533?

**Comment by austinchiang** [comment]
Yes looks like it

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by austinchiang** [comment]
ci:forward:force

**Comment by austinchiang** [comment]
ci:forward:force

**Comment by austinchiang** [comment]
ci:forward:force

**Pull request closed without merging**